example/src/test/scala/monocle/function/AtExample.scala

```diff
@@ -34,4 +38,21 @@ class AtExample extends MonocleSuite {
     (Set(1, 2, 3) applyLens at(2) set None)     shouldEqual Set(1, 3)
   }
 
+  test("at creates a Lens from Int to one of its bit") {
+    (3 applyLens at(0: IntBits) get) shouldEqual true   // true  means bit is 1
```

Comment:
yes I think the ascription is unfortunately needed because the type inference infer needs to know both the type of the source and the type of the index. We know the source by using `applyLens` but the index cannot be infer to the right type and so the `auto._` is not used :(